### DISCUSSION TITLE
How that work?

### DISCUSSION THREAD

**Discussion by djskual** [XorTroll/emuiibo#252] [General]
Hello, i'm perhaps a little bit noob, but all is configured on my sd, but can't get it to emulate some amiibo.

I boot TLOZTOTK, i activate amiibos in game, i made the L+down+R3, the tesla menu append, i open emuiibo 1.0.0, but game are not intercepted. and when i try to use newly made virtual amiibo with latest emutool, nothing append.

Please, a little bit of help.

Thanks.

**Comment by lookye** [comment]
Please read the readme of emuiibo here on github carefully. If you need more help, please join the linked discord server from the top in the readme.
